<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RicercaFumettiAcquistati</class>
 <widget class="QWidget" name="RicercaFumettiAcquistati">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ricerca fumetti</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>681</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerca in base al barcode e seleziona i fumetti che il cliente vuole acquistare:</string>
   </property>
  </widget>
  <widget class="QListView" name="lista_fumetti">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>691</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Barcode:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_barcode">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cerca">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerca</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_inserisci">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserisci</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_completa">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>60</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Completa acquisto</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
